Source/FuseJS/Environment.uno

```diff
@@ -64,13 +66,70 @@ namespace FuseJS
 			AddMember(new NativeProperty<bool, bool>("dotnet", defined(DOTNET))); // Undocumented for testing use only
 			AddMember(new NativeProperty<bool, bool>("host_mac", defined(HOST_MAC))); // Undocumented for testing use only
 			AddMember(new NativeProperty<string, string>("mobileOSVersion", GetMobileOSVersion));
+			AddMember(new NativeProperty< string, object >("locale", GetCurrentLocale));
```

Comment:
Style: Please drop spaces inside `<` and `>`.

Reply:
oops! fix on the way